docs/Guides-Renderer.md

```diff
@@ -0,0 +1,168 @@
+---
+id: guides-renderer
+layout: docs
+category: Guides
+permalink: docs/guides-renderer.html
+next: guides-ready-state
+---
+
+So far, we've covered two pieces that each contribute to declaring data:
+
+ - **Relay.Route** lets us declare query roots.
+ - **Relay.Container** lets components declare fragments.
+
+To use these pieces to construct a full-fledged GraphQL query that we can send to the server to fetch data, we need to use the **Relay.Renderer**.
+
+## Component and Route
+
+**Relay.Renderer** is a React component that, given a `Container` and a `route` or `queryConfig`, attempts to fulfill the data required in order to render an instance of `Container`.
+
+The container created using `Relay.createContainer` must be supplied via the `Container` prop, and the query configuration that conforms to the shape of a `RelayQueryConfig` must be supplied via the `queryConfig` prop.
+
+```
+ReactDOM.render(
+  <Relay.Renderer
+    Container={ProfilePicture}
+    queryConfig={profileRoute}
+  />,
+  component
+);
+```
+
+When the **Relay.Renderer** above is rendered, Relay will construct a query and send it to the GraphQL server. As soon as all required data has been fetched, `ProfilePicture` will be rendered. Props with fragments will contain data that was fetched from the server.
+
+If either `Container` or `queryConfig` ever changes, **Relay.Renderer** will immediately start attempting to fulfill the new data requirements.
+
+**Relay.Renderer** renders the loading state whenever it cannot immediately fulfill data needed to render. This often happens on the initial render, but it can also happen if either `Container` or `queryConfig` changes.
```

Comment:
“The loading state” here is a bit confusing, since that's not actually an entity in the system. As you've noted, the default is to render `null` or the previous view. Maybe merge these two paragraphs?

Also, I think you meant for this text to be below the heading.